app/fissile.go

```diff
@@ -601,6 +584,32 @@ func (f *Fissile) GetDiffConfigurationBases(releasePaths []string, cacheDir stri
 	return getDiffsFromReleases(f.releases)
 }
 
+// Since each job processes only its own spec, inject the hcf-release's
+// patches-properties pseudo-job's spec into all the other jobs.
+func (f *Fissile) injectPatchPropertiesJobSpec() error {
+	var patchPropertiesJob *model.Job
+	for _, release := range f.releases {
+		if release.Name == "hcf" {
+			for _, job := range release.Jobs {
+				if job.Name == "patch-properties" {
+					patchPropertiesJob = job
+				}
+			}
+		}
+	}
+	if patchPropertiesJob == nil {
+		return nil
+	}
+	for _, release := range f.releases {
+		for _, job := range release.Jobs {
+			if release.Name != "hcf" && job.Name != "patch-properties" {
```

Comment:
Does `if job != patchPropertiesJob` work? (I'm actually not sure; I don't _think_ there's a copy, because `release.Jobs` is a `[]*Job`, but it needs confirming…)